server/src/infra/repositories/typesense.repository.ts

```diff
@@ -313,8 +313,8 @@ export class TypesenseRepository implements ISearchRepository {
         {
           collection: assetSchema.name,
           q: '*',
-          vector_query: `smartInfo.clipEmbedding:([${input.join(',')}], k:100)`,
-          per_page: 100,
+          vector_query: `smartInfo.clipEmbedding:([${input.join(',')}], k:1000)`,
+          per_page: 1000,
```

Comment:
https://typesense.org/docs/0.25.1/api/search.html#pagination-parameters max is 250